### ISSUE TITLE
Cannot tab to Close button

### ISSUE THREAD

**Issue by bgurney** [getsentry/sentry-docs#1382]
There is an accessibility issue for keyboard access in that you cannot tab to the Close button (or use Esc key to close it). This means that the dialog is stuck open and users cannot use site navigation to go anywhere else.

**Comment by cameronmcefee** [comment]
@bgurney What dialogue/close button are you referring to?

**Comment by bgurney** [comment]
Sorry, the User Feedback dialog that can be used on error pages for collecting additional information.
